kevm-js/kevm.js

```diff
@@ -24,43 +22,84 @@ const request = require('request-promise-native')
 
+var PRIVATEKEY;
+var CONTRACTADDRESS;
+
 process.on('unhandledRejection', err => {
   console.log(err);
 });
 
 const providerUrl = 'https://kevm-testnet.iohkdev.io:8546'
 //const providerUrl = 'http://localhost:3000'
 
+// ******************************
+// Step 1 - look for existing privateKey and contractAddress in local config file
+// ******************************
+
+var nconf = require('nconf');
+nconf.use('file', { file: './myData.json' });
+nconf.load();
+if (nconf.get('privateKey') != undefined ) {
+	PRIVATEKEY = nconf.get('privateKey');
+    console.log('Existing personal wallet privateKey found in myData.json');
+} else {
+    console.log('No personal wallet privateKey found in myData.json');
+}
+if (nconf.get('contractAddress') != undefined ) {
+	CONTRACTADDRESS = nconf.get('contractAddress');
+    console.log('Existing deployed contractAddress found in myData.json');
+} else {
+    console.log('No deployed contractAddress found in myData.json');
+}
+
+  
 const run = async () => {
   const web3 = new Web3(providerUrl)
 
   // ******************************
-  // Step 1 - create an account
-  // To create an account, you need a random private key. This portion of the 
-  // script will create a random key string that can simply be added to the top
-  // of the script so it can be re-run with the same account.
+  // Step 2 - try using the account, and if it fails create a new one
+  // To create an account, you need a random generated private key. 
+  // This portion of the script will try create a new random key string
+  // The privateKey is stored in local config file for further operations
   // ******************************
 
   try {
     var account = web3.eth.accounts.privateKeyToAccount(PRIVATEKEY)
   } catch (err) {
-    const rand = crypto.randomBytes(32).toString('hex')
-    console.log("No private key - generating random key.")
-    console.log("- add the following line to the top of the script and re-run:")
-    console.log("const PRIVATEKEY = '0x" + rand + "'");
-    process.exit()
-  }
-  const res = web3.eth.accounts.wallet.add(account)
+    const rand = crypto.randomBytes(32).toString('hex');
+	account = web3.eth.accounts.privateKeyToAccount("0x" + rand + "");
+	nconf.set('privateKey', '0x'+rand+'');
+	nconf.save(function (err) {
+		if (err) {
+			console.error(err.message);
+			return;
+		} else {
+			console.log("A new random PRIVATEKEY was generated and stored in ./myData.json");
+			console.log("This key will be used now and for future operations");
+		}
+	  });
+  } 
+  var res = web3.eth.accounts.wallet.add(account);
 
   // ******************************
-  // Step 2 - fund the account
+  // Step 3 - fund the account
   // If the account balance is zero, here we request test tokens from the IOHK faucet
   // and wait until the account is funded.
   // ******************************
 
-  console.log("Account = " + account.address)
+  //console.log("Account = " + account.address)
```

Comment:
Why we don't want to print out the account address?

Reply:
I would rather then vote for printing the wallet address by default, since people would probably want to copy it to the explorer. Address is a public info and may be **very** useful for later debugging and diagnostics.

About points:

1, 2 - totally agree

3 - For unix-like systems there's already the [`tee`](https://stackoverflow.com/questions/418896/how-to-redirect-output-to-a-file-and-stdout) tool that does exactly this. On windows there also seem to be [a solution for this](https://stackoverflow.com/questions/796476/displaying-windows-command-prompt-output-and-redirecting-it-to-a-file). If there's a nice way to do that from inside the script - then I would probably vote **for** it, but we at least should consider pro\cons and cost\profit of having a functionality for something that is already doable :)